libs/scripts/get_video_watching_durations.py

```diff
@@ -0,0 +1,92 @@
+import datetime
+import json
+import psycopg2
+
+
+# Gets datetime object from log 'time' string
+def parse_datetime(log_timestamp):
+    dt = log_timestamp.split("T")
+    date = dt[0].split("-")
+    time = dt[1].split(".")[0].split(":")
+    return datetime.datetime(int(date[0]), int(date[1]), int(date[2]), hour=int(time[0]), minute=int(time[1]), second=int(time[2]))
+
+
+# Prints total time of video-watching for every user (as a table) into console
+def print_result_to_stdout(user_times):
+    print("username | time (sec)")
+    for username in user_times:
+        print(username, "|", user_times[username].total_seconds())
+
```

Comment:
When I'm asking to add system-logs, then I'm talking about the scripts for installation soft, for scripts, that extracts the data, for the scripts, that clean working directory. 

Please, take a look at this scripts, that should log the out to both console and system-logs folder: https://github.com/mandarinSh/OpenPolyEdu/tree/master/bin 

At the same time the analytic scripts, that you have just provided is not supposed to print to system-out (for the time being). Probably it is a good idea to have a debug log be printed, but not a result itself.